app/(admin)/admin/notifications/page.test.tsx

```diff
@@ -1,14 +1,13 @@
-// /Users/ryanfurrer/Developer/GitHub/gridiron-survivor/app/(admin)/admin/notifications/page.test.tsx
```

Comment:
Missing 
```
// Copyright (c) Gridiron Survivor.
// Licensed under the MIT License.
```

At the top here, please re-add it back in here

Reply:
How in the hell did this even get removed?